versioned_sidebars/version-3.9-sidebars.json

```
@@ -10,28 +10,60 @@
       "label": "About ScalarDL",
       "collapsible": true,
       "items": [
-        "overview",
-        "design",
-        "implementation",
-        "requirements",
-        "roadmap"
+        {
+          "type": "doc",
+          "id": "overview",
+          "label": "Overview"
+        },
+        {
+          "type": "doc",
+          "id": "design",
+          "label": "Design"
+        },
+        {
+          "type": "doc",
+          "id": "implementation",
+          "label": "Implementation"
+        },
+        {
+          "type": "doc",
+          "id": "requirements",
+          "label": "Requirements"
+        },
+        {
+          "type": "doc",
+          "id": "roadmap",
+          "label": "Roadmap"
+        }
       ]
     },
     {
       "type": "category",
       "label": "Getting Started",
       "collapsible": true,
       "items": [
-        "getting-started",
-        "getting-started-auditor"
+        {
+          "type": "doc",
+          "id": "getting-started",
+          "label": "Run a Contract Through ScalarDL Ledger"
+        },
+        {
+          "type": "doc",
+          "id": "getting-started-auditor",
+          "label": "Run a Contract Through ScalarDL Ledger and ScalarDL Auditor"
```

```suggestion
          "label": "Run a Contract Through ScalarDL Auditor mode"
```
Same as `sidebars.js`.

---

Addressed, as mentioned in the https://github.com/scalar-labs/docs-scalardl/pull/597/files#r1897305521 comment thread.